openeogeotrellis/vectorcube.py

```diff
@@ -139,25 +165,38 @@ def _get_run_udf_udf_data_callback(
         :param context:
         :return:
         """
-        def callback(data: pandas.DataFrame):
+        import pyspark.pandas
+        def callback(id_bands) -> pyspark.pandas.DataFrame:
             # Get current feature index and drop whole column
-            feature_index = data["feature_index"].iloc[0]
-            feature_data = data.drop("feature_index", axis=1)
+            #feature_index = data["feature_index"].iloc[0]
+            #feature_data = data.drop("feature_index", axis=1)
             # TODO: We assume here that the `date` column already has parsed dates (naive, without timezone info).
             #       At the moment `pyspark.pandas.read_csv` seems to parse dates automatically
             #       (as pandas timestamp) even-though the docs states otherwise
             #       also see https://issues.apache.org/jira/browse/SPARK-40934
-            feature_data = feature_data.set_index("date")
-            feature_data.index = feature_data.index.strftime("%Y-%m-%dT%H:%M:%SZ")
+            #if "date" in feature_data.columns:
+            #    feature_data = feature_data.set_index("date")
+            #feature_data.index = feature_data.index.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
can't we remove these lines of code instead of commenting out?  What is the use of keeping them?